### PR TITLE
Added scaling for nozzle and data generation app

### PR DESCRIPTION
**Pull request by chaitanyaphalak**

### CODE REVIEW

File: ci/splunk_nozzle_manifest.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+applications:
+  - name: splunk-firehose-nozzle
+    memory: 512M
+    instances: 2
```
Comment (luckyj5) [inline ci/splunk_nozzle_manifest.yml]: Can we use a variable instead of hardcoded value to set the number of instances? 
@chenziliang

Comment (chenziliang) [inline ci/splunk_nozzle_manifest.yml]: I think we should use {{params}} format for all env. For instances, memory etc, i am good to hard code for now

File: ci/splunk_nozzle_manifest.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    buildpack: go_buildpack
+    env:
+      GOPACKAGENAME: main
+      API_ENDPOINT:
+      API_USER:
+      API_PASSWORD:
```
Comment (chenziliang) [inline ci/splunk_nozzle_manifest.yml]: Can you explore if we can use {{api_password}} format in yml and use another yml file to populate this one ?

Comment (luckyj5) [inline ci/splunk_nozzle_manifest.yml]: Can we use env variables to set these numbers for now and avoid using hard coded value? 
I'll look into how can we use {{params}} format and reference to another yml file.

**Pull request merged**